bin/as-recon-discover-services.tengo

```diff
@@ -0,0 +1,195 @@
+#! /usr/bin/env arsenic
```

Comment:
since we are adding special arsenic stuff. should we change the extension from `.tengo` to `.as` or `.astengo`

Reply:
Maybe. I kept it `.tengo` simply to get the syntax highlighting within vscode using the tengo extension. I believe I can update my vscode settings to associate `.as` or `.astengo` or whatever we decide to tengo so I am fine with changing the extension